earth_enterprise/src/fusion/autoingest/sysman/khAssetManager.cpp

```diff
@@ -574,7 +574,10 @@ khAssetManager::NotifyVersionStateChange(const std::string &ref,
 {
   // assert that we're already locked
   assert(!mutex.TryLock());
-
+  
+  notify(NFY_VERBOSE, "Set pendingStateChanges[ref]: %s to state: %s", 
+         ToString(pendingStateChanges[ref]).c_str(), 
```

Comment:
ToString(pendingStateChanges[ref]).c_str() is printing as a blank when I test this.  

> [2018-09-24 11:08:10] Verbose: Calling theAssetManager.NotifyVersionStateChange(Tutorial/Resources/Imagery/other.kiasset/product.kia?version=1, Queued)
> [2018-09-24 11:08:10] Verbose: Set pendingStateChanges[ref]:  to state: Queued

I would also probably update this notify so it says the actual ref name instead of ref

Reply:
So I see the blank values, but sometimes it will return Blocked. Not really sure what causes this, but I have had it return an expected value.